composer.json

```diff
@@ -124,13 +124,13 @@
         "drupal/block_field": "1.0.0-rc4",
         "drupal/config_modify": "^1",
         "drupal/config_update": "1.7",
-        "drupal/core": "~9.5",
-        "drupal/core-composer-scaffold": "~9.5.3",
+        "drupal/core": "9.5.10 as 10.1.3",
+        "drupal/core-composer-scaffold": "9.5.10 as 10.1.3",
```

Comment:
This is a no-go unfortunately. It suggests to composer that Drupal 10.1 is present which might cause libraries to be installed for downstream users that expect Drupal 10.1 to be present, but we're actually providing Drupal 9.5 which would break their site.

Reply:
As discussed, this was just temporary test and this is already deprecated code :)